src/docfx/lib/git/GitUtility.cs

```diff
@@ -237,28 +237,31 @@ private static string Execute(string cwd, string commandLineArgs, bool stdout =
 
             try
             {
-                return ProcessUtility.Execute("git", commandLineArgs, cwd, stdout, secrets);
+                return ProcessUtility.Execute("git", commandLineArgs, cwd, stdout, secret);
             }
             catch (Win32Exception ex) when (ProcessUtility.IsExeNotFoundException(ex))
             {
                 throw Errors.System.GitNotFound().ToException(ex);
             }
         }
 
-        private static (string cmd, string[] secrets) GetGitCommandLineConfig(string url, PreloadConfig config)
+        private static (string cmd, string? secret) GetGitCommandLineConfig(string url, PreloadConfig config)
```

Comment:
`return default` produces `(null, null)`
```suggestion
        private static (string? cmd, string? secret) GetGitCommandLineConfig(string url, PreloadConfig config)
```

Reply:
Updated, L252 already broke it

Reply:
Yep. Better late than never.